daphne/src/auth.rs

```diff
@@ -12,60 +12,71 @@ use crate::{
     DapError, DapRequest,
 };
 use async_trait::async_trait;
-use prio::codec::Decode;
 use serde::{Deserialize, Serialize};
-use std::io::Cursor;
 
 /// A bearer token used for authorizing DAP requests as specified in draft-ietf-ppm-dap-02.
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct BearerToken(String);
```

Comment:
Reviewer note: For whatever reason, the manner in which serde encodes this struct triggers undefined behavior in miniflare. The new format is tolerated.